ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/Whispers/WhisperClient.cs

```diff
@@ -67,6 +68,27 @@ public Task<bool> ConfirmAsync(WhisperConfirm message, CancellationToken cancell
             return new StreamingCall<WhisperFormStreamResponse, IWhisperFormResponse>(call, loggedParser);
         }
 
+        public IStreamingCall<IWhisperDisambiguationResponse> DisambiguationAsync(WhisperDisambiguation message, CancellationToken cancellationToken = default)
+        {
+            var request = Builder.BuildRequest(message, CreateSession());
+            var call = Client.WhisperDisambiguation(request, new CallOptions(cancellationToken: cancellationToken));
+            var loggedParser = LoggedParser<Proto.WhisperDisambiguationStreamResponse, IWhisperDisambiguationResponse>(response =>
+                Parser.ParseResponse(response)); 
+            return new StreamingCall<WhisperDisambiguationStreamResponse, IWhisperDisambiguationResponse>(call, loggedParser);
+        }
+
+        /* 
+        Need to look into the parser
```

Comment:
What needs to be checked?

Reply:
Nothing, that code block needs to be removed